README.md

```diff
@@ -38,33 +38,47 @@ Sends a deployment complete message to a Slack channel when [Trellis](https://gi
 
 ## Installation
 
-Add this role to `requirements.yml`:
+Add this role to `requirements.yml` or `galaxy.yml` in newer versions of Trellis:
 ```yaml
-# requirements.yml
-- src: https://github.com/ItinerisLtd/trellis-slack-webhook-notify-during-deploy
-  version: 0.1.0 # Check for latest version!
+# requirements.yml / galaxy.yml
+- name: slack-notify
+  src: https://github.com/ItinerisLtd/trellis-slack-webhook-notify-during-deploy
+  version: 0.3.0 # Check for latest version!
 ```
 
 Run the command:
 ```bash
 ➜ ansible-galaxy install -r requirements.yml --force
 ```
 
+With newer versions of Trellis, run:
+```bash
+➜ ansible-galaxy install -r galaxy.yml --force
+```
+
+
 ## Role Variables
 
-Add this role to the [`deploy_after` hook](https://roots.io/trellis/docs/deploys/#hooks):
+Add these roles`deploy_before` & `deploy_finalize_after` to the deploy [hooks](https://roots.io/trellis/docs/deploys/#hooks):
```

Comment:
```suggestion
Add these roles`deploy_before` & `deploy_after` to the deploy [hooks](https://roots.io/trellis/docs/deploys/#hooks):
```

Reply:
Out of curiosity, what's the reason for using `deploy_after` instead of the later hook `deploy_finalize_after`? I'd assume we'd want the success message to be on the latest hook available?

Reply:
I think `deploy_after` is later than `deploy_finalize_after`. Am I missing something?

See: https://github.com/roots/trellis/blob/f11a7cefbc978831d1ff6b4156298e618ff2ff80/roles/deploy/tasks/main.yml

Reply:
haha ah of course it is, my bad - when testing I was using `deploy_build_after`. Cool will switch those over :+1: